Mathlib/MeasureTheory/Integral/RieszMarkovKakutani.lean

```diff
@@ -370,12 +370,12 @@ section RieszMeasure
 variable [T2Space X] [LocallyCompactSpace X] [MeasurableSpace X] [BorelSpace X]
 
 /-- `rieszContent` is promoted to a measure. -/
-def rieszMeasure := (MeasureTheory.Content.measure (rieszContent Λ))
+def rieszMeasure := (rieszContent Λ).measure
 
 lemma leRieszMeasure_Compacts {f : C_c(X, ℝ≥0)} (hf : ∀ (x : X), f x ≤ 1) {K : Compacts X}
-    (h : tsupport f ⊆ K) : ENNReal.ofReal (Λ f) ≤ (μ Λ) K := by
-  simp only [μ, MeasureTheory.Content.measure_eq_content_of_regular (rieszContent Λ)
-    (rieszContentRegular Λ)]
+    (h : tsupport f ⊆ K) : ENNReal.ofReal (Λ f) ≤ (rieszMeasure Λ) K := by
```

Comment:
```suggestion
    (h : tsupport f ⊆ K) : (Λ f).ofReal ≤ rieszMeasure Λ K := by
```

Reply:
sorry, I don't remember why I used `ofReal`, while `Λ f` is `NNReal`. I use `ofNNReal`. The dot notation does not work